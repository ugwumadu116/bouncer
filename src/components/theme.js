import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
