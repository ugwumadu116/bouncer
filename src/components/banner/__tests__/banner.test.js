import React from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import theme from "../../theme";
import Banner from "..";

it("renders correctly when called ", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
        <Banner/>
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
