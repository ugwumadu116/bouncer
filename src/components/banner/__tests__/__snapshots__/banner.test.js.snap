// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<div
  className="vis-hidden"
>
  <div
    className="sc-bxivhb gFwdHb"
  >
    <section
      className="sc-ifAKCX bJMAHJ"
    >
      <div
        className="sc-EHOje hTTwXQ"
      >
        <div>
          <p
            className="sc-bdVaJa hXIHfI"
          >
            iPhone 6 Plus
          </p>
        </div>
        <div>
          <p
            className="sc-bdVaJa dAMGmB"
          >
            Performance and design. Taken right to the edge.
          </p>
        </div>
        <div>
          <p
            className="sc-bdVaJa egjVsa"
          >
            <span
              className="border-1"
            >
              SHOP NOW
            </span>
          </p>
        </div>
      </div>
      <div
        className="sc-bZQynM dzMect"
      >
        <img
          alt="Iphone"
          src="pic.png"
        />
      </div>
    </section>
  </div>
</div>
`;
